jcl/src/java.base/share/classes/java/lang/Object.java

```diff
@@ -130,6 +130,11 @@ protected void finalize () throws Throwable {
  * @see	#equals
  */
 public int hashCode() {
+	/*[IF INLINE-TYPES] */
```

Comment:
@hzongaro Does the JIT recongnize these methods? do we need coordinated changes?

Reply:
It doesn't look like the JIT recognizes `Object.hashCode()`, but it does recognize `System.identityHashCode()`, `J9VMInternals.identityHashCode()` and `J9VMInternals.fastIdentityHashCode()`.  So it might be beneficial for the JIT to recognize `J9VMInternals.valueHashCode()` and likely `Class.isValue()` as well.

I don't think a co-ordinated change is required, as correctness won't be affected.